switchbot_ros/msg/CommandTopic.msg

```diff
@@ -0,0 +1,4 @@
+string nickname
+string command
+int16 times
```

Comment:
please add comment to all values like https://github.com/ros/common_msgs/blob/noetic-devel/sensor_msgs/msg/CameraInfo.msg